index.html

```diff
@@ -190,6 +190,42 @@ <h3 class="Skill">Skills</h3>
             </div>
 
         </section>
+        <footer>
+            <section class="form">
+                <p class="para1">
+            I'm always<br> interested in hearing <br> about new projects, <br> so if you'd like to <br>chat please get in
+                </p>
+                <form action="/action_page.php" class="form_div">
```

Comment:
- [ ] Kindly make sure you are using [Formspree service](https://formspree.io/html) to collect the data. Please sign up and use that link.